site/_config.yml

```diff
@@ -10,7 +10,7 @@ twitter:
   card: summary
 vm_logo: vm-logo.png
 gh_repo: https://github.com/projectcontour/contour
-markdown: redcarpet
+markdown: kramdown
```

Comment:
Is there anything we're missing out on by going with kramdown vs redcarpet? I just want to make sure we've covered that.

Reply:
I'm in the process of validating it. now that the docs update has been merged. There is no support for Redcarpet in Jekyll 4.